pkg/kinflate/resource/configmap.go

```diff
@@ -130,3 +130,15 @@ func addKV(m map[string]string, kv kvPair) error {
 	m[kv.key] = kv.value
 	return nil
 }
+
+func NewFromConfigMapList(cmList []manifest.ConfigMap, loader loader.Loader) ([]*Resource, error) {
+	allResources := []*Resource{}
+	for _, cm := range cmList {
+		res, err := NewFromConfigMap(cm, loader)
+		if err != nil {
+			return nil, err
+		}
+		allResources = append(allResources, res)
+	}
+	return allResources, nil
+}
```

Comment:
If we end up calling NewFromPaths.... for resources then we should change this to
`func NewFromConfigMaps(loader..., cms...)`
Move loader as the first arg in all the methods.

Reply:
Only one loader though :-)

Reply:
yes, only one loader :)